src/hotkey/hotkey_item.cpp

```diff
@@ -157,6 +157,7 @@ bool hotkey_base::matches(const SDL_Event &event) const
 void hotkey_base::save(config& item) const
 {
 	item["command"] = get_command();
+	item["disabled"] = is_disabled() ? config::attribute_value::s_true : config::attribute_value::s_false;
```

Comment:
don't do this, s_true and s_false are just meant for configs internal use. i don't even know why they are public.